components/tidb_query_datatype/src/codec/mysql/time/mod.rs

```diff
@@ -2737,8 +2737,8 @@ mod tests {
     #[test]
     fn test_convert_to_f64() {
         let cases = vec![
-            ("2012-12-31 11:30:45.123456", 4, 20121231113045.1235f64),
```

Comment:
I recommend suppressing this lint. (Same to the following)

Reply:
Fixed